src/ai/backend/client/cli/session.py

```diff
@@ -950,13 +938,23 @@ def watch(session_name_or_id: str, owner_access_key: str, scope: str, max_wait:
                 print_fail('No matching items.')
             sys.exit(4)
 
+        kernel_cancelled: Final[str] = 'kernel_cancelled'
+        kernel_creating: Final[str] = 'kernel_creating'
+        kernel_started: Final[str] = 'kernel_started'
+        kernel_terminated: Final[str] = 'kernel_terminated'
+        kernel_terminating: Final[str] = 'kernel_terminating'
+        session_failure: Final[str] = 'session_failure'
+        session_started: Final[str] = 'session_started'
+        session_success: Final[str] = 'session_success'
+        session_terminated: Final[str] = 'session_terminated'
```

Comment:
In Python, it is recommended to use uppercase names for constants. I think this may be defined in the top-level scope of the module or a new module.